msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -86,11 +86,43 @@ See the following video to learn how to send proactive message from bots:
 
 Sending proactive messages to the users is an effective way to communicate with your users. However, from the user's perspective, the message appears unprompted. If there's a welcome message, it will be the first time that they’ve interacted with your app. It's important to use this functionality and provide the complete information to the user to understand the purpose of this message.
 
+> [!NOTE]
+>
+> * You can send a proactive message to the bot to know if a user has blocked or uninstalled a bot. If the bot is blocked or uninstalled, Teams returns a 403 response with a `subCode` **MessageWritesBlocked** indicating that the chat thread is blocked from message writes.
```

Comment:
What does `blocked from message writes` mean? Simplify it? 

`MessageWritesBlocked` should be in backticks and not in bold.

Reply:
updated that the user can't receive any messages sent by the bot.